README.md

```diff
@@ -1,47 +1,47 @@
-# OpenEBS CSI Driver
+# OpenEBS CStor CSI Driver
 
-CSI driver implementation for OpenEBS storage engines. 
+CSI driver implementation for OpenEBS CSTOR storage engine.
 
 ## Project Status
 
 This project is under active development and considered to be in Alpha state.
 
-The current implementation only supports provisioning, de-provisioning and expansion of cStor Volumes. 
+The current implementation only supports provisioning, de-provisioning and expansion of cStor Volumes.
 
 ## Usage
 
 ### Prerequisites
 
-Before setting up OpenEBS CSI driver make sure your Kubernetes Cluster 
+Before setting up OpenEBS CStor CSI driver make sure your Kubernetes Cluster
```

Comment:
Add one more note or pre-req .. about CStor CSI Driver operates on the cStor Pools created using the new Schema called CSPC. Provide a link to the design to this new CSPC schema.